templates/colors/literals-swift3.stencil

```diff
@@ -11,10 +15,12 @@
   {% if enumName != 'UIColor' %}{{accessModifier}} enum {{enumName}} { }{% endif %}
 #endif
 
+{% if avoidSwiftLintDisableAll %}
```

Comment:
It's annoying that the templates will get noisier with these checks. But I'm not sure if there's a better way to do this.

We could keep the unneeded `// swiftlint:disable specific rule`, and only have the initial check to add the `disable all` line.